workflow-templates/test-package.yaml

```diff
@@ -12,12 +12,13 @@ env:
 
 jobs:
   # Check code formatting and static analysis on a single OS (linux)
-  # against Dart stable and dev.
+  # against Dart dev.
   analyze:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        sdk: [dev, stable]
+        # Add stable if the package should also be tested on stable
+        sdk: [dev]
```

Comment:
Why remove stable? That is what pub.dev will score on.

Reply:
It looked like most of the packages just analyze on dev. I believe that's because we currently have the null-safe versions on master for many packages and those don't analyze successfully until 2.12 hits stable.

Reply:
We should model doing it on stable – even if most folks can't now

Reply:
Done, stable and dev are both enabled by default now.